### DISCUSSION TITLE
Ability to format the text on Secure Notes

### DISCUSSION THREAD

**Discussion by zzahkaboom24** [dani-garcia/vaultwarden#3069] [Ideas]
Can the Secure Notes functionality be extended to support formatting?
I would like to be able to organize my Secure Notes slightly more with formatting options like bold, underlined, etc. but I cant do so.
Or is this already a thing that I am not aware of?
Upon searching for similar discussions, I havent found a similar discussion talking about this, but If I am wrong. This discussion can be closed

**Comment by BlackDex** [comment]
That is something for the Bitwarden community forums i think.
That all is done client side. And Vaultwarden doesn't maintain the clients.

**Reply by zzahkaboom24** [reply]
Ah okay, then the discussion can be closed, as I did read on these forums and there have been posts even up to 2019 so i dont think there will be something coming soon, ill close the discussion and will continue using secure notes like they are now but in my own way of organization, thanks for answering tho.
